README.rst

```diff
@@ -130,6 +130,19 @@ These effect how we display the Table of Contents in the side bar. You can read
 * ``includehidden`` Bool. Specifies if the sidebar includes toctrees marked with the `:hidden:` option
 * ``titles_only`` Bool. If True, removes headers within a page from the sidebar.
 
+HTML Context Configuration
+--------------------------
+
+.. code:: python
+
+    html_context = {
+        'pageview_mode': ''
```

Comment:
This needs a prefix to clarify that it sets the way links to the _version control system_ are formatted, rather than something related to RTD or the theme. `github_pageview_mode` is not suitable because the setting is used for GitHub, GitLab and BitBucket. What about `vcs_pageview_mode`?
I think this also needs documentation for `bitbucket_user` etc.